src/components/PersonTable/PersonTable.tsx

```diff
@@ -0,0 +1,60 @@
+import { Link, useParams } from 'react-router-dom';
+import classNames from 'classnames';
+import { Person } from '../../types';
+
+type Props = {
```

Comment:
use interface prefer. 
Types have unique features but the interface is best for scaling.